invenio_app_ils/ui/backoffice/src/pages/UserDetails/state/__tests__/actions.js

```diff
@@ -0,0 +1,87 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import MockAdapter from 'axios-mock-adapter';
+import { http } from 'common/api';
+import * as actions from '../actions';
+import { initialState } from '../reducer';
+import * as types from '../types';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+const usersBaseUrl = 'https://127.0.0.1:5000/api/users';
+
+describe('Users actions', () => {
+  let store;
+  beforeEach(() => {
+    store = mockStore(initialState);
+    store.clearActions();
+  });
+
+  describe('Fetch users details tests', () => {
+    let mock;
+    let response = {};
+    beforeEach(() => {
+      mock = new MockAdapter(http);
+      mock.onGet(`${usersBaseUrl}/1`).reply(() => {
+        return new Promise((resolve, reject) =>
```

Comment:
I would really avoid to do this, I thought I have already removed it everywhere.
I would suggest to mock the `getRecord` method and not the http call, it is much easier. On top of that, Redux action does not know that there is axios behind, it knows that calling the method `getRecord` it will receive a Promise.

Reply:
I still dislike this very much 😄, IMHO overcomplicated